src/main/groovy/edu/wpi/first/gradlerio/wpi/toolchain/WPIToolchainPlugin.groovy

```diff
@@ -199,6 +207,53 @@ class WPIToolchainPlugin implements Plugin<Project> {
             toolChainRegistry.registerDefaultToolChain('roborioGcc', WPIRoboRioGcc)
         }
 
+        @BinaryTasks
+        void createNativeStripTasks(final ModelMap<Task> tasks, final ProjectLayout projectLayout, final NativeBinarySpec binary) {
+            def project = (Project) projectLayout.projectIdentifier
```

Comment:
Our current convention is to use the ProjectWrapper as an extension, since with `projectIdentifier` we're assuming a cast.  https://github.com/wpilibsuite/GradleRIO/blob/backtrack2019/src/main/groovy/edu/wpi/first/gradlerio/wpi/toolchain/WPIToolchainPlugin.groovy#L190